mlflow/utils/file_utils.py

```diff
@@ -694,81 +697,54 @@ def parallelized_download_file_using_http_uri(
     Returns a dict of chunk index : exception, if one was thrown for that index.
     """
 
-    def run_download(range_start, range_end):
-        template = """
+    def run_download(chunk: _Chunk):
+        try:
+            subprocess.run(
+                [
+                    sys.executable,
+                    download_cloud_file_chunk.__file__,
+                    "--range-start",
+                    str(chunk.start),
+                    "--range-end",
+                    str(chunk.end),
+                    "--headers",
+                    json.dumps(headers or {}),
+                    "--download-path",
+                    download_path,
+                    "--http-uri",
+                    http_uri,
+                ],
+                text=True,
+                check=True,
+                capture_output=True,
+                timeout=MLFLOW_DOWNLOAD_CHUNK_TIMEOUT.get(),
+                env=env,
+            )
+        except (TimeoutExpired, CalledProcessError) as e:
+            raise MlflowException(
+                f"""
 ----- stdout -----
-{stdout}
+{e.stdout.strip() or 'N/A'}
 
 ----- stderr -----
-{stderr}
+{e.stderr.strip() or 'N/A'}
```

Comment:
Nit: If `e.stdout` and `e.stderr` are undefined, can we just leave it blank instead of printing `N/A`?

Reply:
I think they should be defined: https://docs.python.org/3/library/subprocess.html

Reply:
got it.